web-ui/src/api/team.js

```diff
@@ -3,18 +3,16 @@ import { resolve } from "./api.js";
 const teamUrl = "/services/team";
 const teamMemberUrl = "/services/team/member";
 
-export const getAllTeamMembers = async (cookie) => {
+export const getAllTeamMembers = async () => {
   return resolve({
-    method: "get",
     url: teamMemberUrl,
     responseType: "json",
-    headers: { "X-CSRF-Header": cookie },
```

Comment:
did this get delete when resolving merge conflicts or this is intentional ?

Reply:
Good catch, thanks